data/formatters/generic.yaml

```diff
@@ -635,10 +635,10 @@ short_message:
 type: 'conditional'
 data_type: 'syslog:ssh:failed_connection'
 message:
-- 'Unsuccessful connection of user: {username}'
+- 'Unsuccessful connection of user: {username} '
 - 'from {address}:'
-- '{port}'
-- 'using authentication method: {authentication_method}'
+- '{port} '
+- 'using authentication method: {authentication_method} '
 - 'ssh pid: {pid}'
 separator: ''
```

Comment:
should we remove the custom separator instead and just use default of space?

Reply:
I think then we'd have a space between IP and port
> ...from 10.0.0.1: 4566 using... 

I'm not too familiar with the formatters, but I'm assuming there's a reason each variable is on its own line. Otherwise, we could combine address:port on one line, then use the default space separator.

Reply:
> I'm not too familiar with the formatters, but I'm assuming there's a reason each variable is on its own line.

mainly historical, was sufficient at the time. I can make a note to enhance the message formatters in this aspect. There is more formatting work in the pipeline

Reply:
more info can be found here: https://plaso.readthedocs.io/en/latest/sources/user/Output-and-formatting.html#message-formatting

but the tl;dr is that the individual string (parts/segment) will be combined using the separator. A string part will be present if the value is available.